src/main/java/org/openelisglobal/qaevent/controller/rest/NonConformingEventsCorrectionActionRestController.java

```diff
@@ -26,12 +26,10 @@
 
 @RestController
 @RequestMapping(value = "/rest")
-public class NonConformingEventsCorrectionActionRestController {
+public class NonConformingEventsCorrectionActionRestController extends ControllerUtills {
```

Comment:
This is a Rest Controller oler so it  should  extend  the` BaseRestController` instead